lib/cli/drive-selection-widget.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+const _ = require('lodash');
+
+const DynamicList = require('./dynamic-list');
+const scanner = require('./drive-scanner');
+const driveComparer = require('../shared/utils').driveComparer;
+const constraints = require('../shared/drive-constraints');
+
+inquirer.registerPrompt('dynamicList', DynamicList);
+
+const driveToChoiceFn = (image) => {
```

Comment:
This function should contain a JSDoc annotation. Also, let's drop the `Fn` prefix.